main/dino-info.rc

```diff
@@ -0,0 +1,25 @@
+1 VERSIONINFO
+FILEVERSION     0,1,0,0
```

Comment:
Various versions in this file are hardcoded, but they should be set at build time.

Reply:
Done